maven-core/src/main/java/org/apache/maven/plugin/ReflectionValueExtractor.java

```diff
@@ -261,22 +265,14 @@ private static Object getPropertyValue(Object value, String property) throws Int
 
         ClassMap classMap = getClassMap(value.getClass());
         String methodBase = Character.toTitleCase(property.charAt(0)) + property.substring(1);
-        String methodName = "get" + methodBase;
         try {
-            Method method = classMap.findMethod(methodName);
-
-            if (method == null) {
-                // perhaps this is a boolean property??
-                methodName = "is" + methodBase;
-
-                method = classMap.findMethod(methodName);
+            for (String prefix : Arrays.asList("get", "is", "to", "as")) {
+                Method method = classMap.findMethod(prefix + methodBase);
+                if (method != null) {
+                    return method.invoke(value, OBJECT_ARGS);
+                }
```

Comment:
Shouldn't this be a separate, preceding PR?

Reply:
No, that's one of the important change to support `toXxx` and `asXxx`.

Reply:
I know, but can be logically done *before* this change, no? But anyway, let's leave this hunk as-is.